### ISSUE TITLE
VTK MooseDocs error if LIBMESH_HAVE_VTK=false

### ISSUE THREAD

**Issue by joshuahansel** [idaholab/moose#11949]
## Rationale
A MooseDocs error occurs when the pre-processor `LIBMESH_HAVE_VTK` is false because `VTKOutput` doesn't get registered, so MooseDocs does not recognize the syntax for `/Outputs/VTK`.

## Description
See above.

## Impact
Fix MooseDocs for when `LIBMESH_HAVE_VTK` is false.

**Comment by joshuahansel** [comment]
@aeslaughter @jwpeterson : It was suggested by @andrsd that the correct solution is to always register `VTKOutput`, but just throw an error in the constructor if the pre-processor is false. Otherwise I think MooseDocs would need to have access to that pre-processor value to see whether to include it in docs.

**Comment by andrsd** [comment]
Just for the record, the problem we are seeing looks like this when we run `./moosedocs.py build --check`:

```
MooseDocs.base.readers: 
ERROR: '/Outputs/VTK' syntax was not recognized.
/path/app/moose/framework/doc/content/source/outputs/VTKOutput.md:9
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
 9│                                                                                                │
10│                                                                                                │
11│!syntax description /Outputs/VTK                                                                │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘

Traceback (most recent call last):
  File "/path/app/moose/python/MooseDocs/base/lexers.py", line 230, in tokenize
    obj = self.buildObject(parent, pattern, info)
  File "/path/app/moose/python/MooseDocs/base/lexers.py", line 314, in buildObject
    obj = super(RecursiveLexer, self).buildObject(parent, pattern, info)
  File "/path/app/moose/python/MooseDocs/base/lexers.py", line 256, in buildObject
    obj = pattern.function(info, parent)
  File "/path/app/moose/python/MooseDocs/base/components.py", line 149, in __call__
    token = self.createToken(info, parent)
  File "/path/app/moose/python/MooseDocs/extensions/command.py", line 114, in createToken
    token = obj.createToken(info, parent)
  File "/path/app/moose/python/MooseDocs/extensions/appsyntax.py", line 187, in createToken
    obj = self.extension.find(self.settings['syntax'], self.NODE_TYPE)
  File "/path/app/moose/python/MooseDocs/extensions/appsyntax.py", line 146, in find
    raise exc(msg, name)
TokenizeException: '/Outputs/VTK' syntax was not recognized.

```

**Comment by aeslaughter** [comment]
I have known about this problem for a long time, but never wanted to tackle it because as mentioned above I needed to look at libmesh_config or something. But, the supplied fix in #11950 works for me, it is simple. Thanks.
